python-import/finders_and_loaders/blacklist_importer.py

```diff
@@ -0,0 +1,15 @@
+# blacklist_importer.py
+
+import sys
+
+BLACKLISTED_MODULES = {"re"}
```

Comment:
In light of recent events, I've seen some people speaking out against the allusions to race in the whitelist/blacklist terminology.

With things as heated as they are in the US, I wouldn't be surprised if someone commented on it.

How do you feel about replacing the word "Blacklisted" with "Banned"?

Reply:
Makes sense. Thanks for picking up on that.

I've updated the file in `materials`. @jaschmitt Should I do the corresponding updates in the article itself, or will you handle that?

- Rename file: `blacklist_importer.py` -> `ban_importer.py`
- Rename constant: `BLACKLISTED_MODULES` -> `BANNED_MODULES`
- Rename class: `BlacklistFinder` -> `BanFinder`
- Rephrase error message: `f"{name!r} is blacklisted"` -> `f"{name!r} is banned"`